### ISSUE TITLE
Encryption upgrade fails to fetch keys from backup

### ISSUE THREAD

**Issue by tctovsli** [element-hq/element-web#13575]
### Description

During first login on riot-web 1.16 (or subsequent "Restore keys from backup" or similar actions), the client fails to fetch keys from the server.
The GUI keeps spinning on "Fetching keys" and nothing happens.

### Steps to reproduce

- Log in with account that has key-backup enabled and member of some encrypted rooms before the new E2EE.
- When prompted to upgrade encryption, enter the account password and press OK
- The wheel spins and nothing happens. In the console of the browser this message is seen: ```restoreWithCachedKey failed: Error: "Couldn't get key"
    restoreKeyBackupWithCache https://riot.midlertidig.online/bundles/9f769ac7a648e2ec9507/vendors~init.js:2```

Log: sent

### Version information

- **Platform**: web (in-browser)

For the web app:

- **Browser**: Firefox 75.0
- **OS**: Windows, Manjaro
- **URL**: Private server, running riot-web version: v1.6.0

**Comment by tctovsli** [comment]
This also happens in the newest riot-desktop on Windows 10.

**Comment by dbkr** [comment]
Trying to repro this but no luck so far. After enetering the account password and clicking, 'Next', you should get a prompt for your key backup passphrase. Are you not getting that prompt?

**Comment by tctovsli** [comment]
No. I was expecting that, but it just asked about account passphrase.

tor. 7. mai 2020, 14:26 skrev David Baker <notifications@github.com>:

**Comment by tctovsli** [comment]
I guess I can solve it by deleting the key backup, but that won't fix the
real issue.

tor. 7. mai 2020, 15:54 skrev Tor Christian Tovslid <tctovsli@gmail.com>:

**Comment by DarwinAwardWinner** [comment]
I'm seeing this same on Mac OS, using an account on matrix.org. I get to this screen:

<img width="589" alt="Screen Region 2020-05-07 at 16 29 15" src="https://user-images.githubusercontent.com/228145/81341847-3ead9b80-9080-11ea-932f-805b49da1764.png">

After entering my password and pressing enter, I come to this screen:

<img width="733" alt="Screen Region 2020-05-07 at 16 31 22" src="https://user-images.githubusercontent.com/228145/81341871-4b31f400-9080-11ea-8c24-0a929b37e141.png">

The spinner spins for a few seconds, and then I end up back at the same exact password entry screen as above, with no error message. I can enter my password again and the same thing will happen -- I just end up back where I started with no explicit indication of either progress or failure.

**Comment by DarwinAwardWinner** [comment]
Also, this might be a separate bug, but if I press "skip" and then "go back", I end up here, which is definitely not back where I was before:

<img width="596" alt="Screen Region 2020-05-07 at 16 40 10" src="https://user-images.githubusercontent.com/228145/81342574-72d58c00-9081-11ea-97c8-306b3383e88d.png">

**Comment by PopeRigby** [comment]
I'm also seeing this on ArchLinux using a [feneas.org](https://feneas.org) account

**Comment by bergmannf** [comment]
For me it was working when setting it up on the `RiotX` Android App, but once I try to set it up on the desktop application (Fedora using the Flatpak) it will go into a loop asking about the account passphrase.

It never asks about the key backup passphrase only the account password.

**Comment by takiainen** [comment]
I've got this same issue, both with riot desktop (linux) and riot web at riot.im/staging. Cannot get in due to "encryption upgrade" failing over and over again.

**Comment by fbodr** [comment]
Same problem here, with Ubuntu 20.04 and Firefox 76. 
It keeps asking the pasword to upgrade the encryption key, but does not accept the password.

**Comment by merrpink** [comment]
Same. Cannot verify new device and cannot read past messages. Spinner just spins and nothing happens.

**Comment by richvdh** [comment]
I don't think this flow exists any more.
